src/main/java/ca/softwarespace/qiyanna/dataaggregator/repositories/MatchRepository.java

```diff
@@ -0,0 +1,118 @@
+package ca.softwarespace.qiyanna.dataaggregator.repositories;
+
+import com.merakianalytics.orianna.types.dto.match.Match;
+import java.util.List;
+import java.util.Optional;
+import org.springframework.data.domain.Example;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.data.mongodb.repository.MongoRepository;
+
+public class MatchRepository implements MongoRepository<Match, String > {
```

Comment:
Not in use atm? as the methods dont really contain anything, maybe best to clean up for now?